src/structure_managers/adaptor_neighbour_list.hh

```diff
@@ -453,20 +453,15 @@ namespace rascal {
      * radius or extends an existing neighbourlist to the next order
      */
     AdaptorNeighbourList(ImplementationPtr_t manager, double cutoff,
-                         bool consider_ghost_neighbours = false);
-
-    AdaptorNeighbourList(ImplementationPtr_t manager, std::tuple<double> tp)
-        : AdaptorNeighbourList(manager, std::get<0>(tp)) {}
-
-    AdaptorNeighbourList(ImplementationPtr_t manager,
-                         std::tuple<double, bool> tp)
-        : AdaptorNeighbourList(manager, std::get<0>(tp), std::get<1>(tp)) {}
+                         const bool & consider_ghost_neighbours = false,
+                         const double & skin = 0.);
 
     AdaptorNeighbourList(ImplementationPtr_t manager,
                          const Hypers_t & adaptor_hypers)
         : AdaptorNeighbourList(
               manager, adaptor_hypers.at("cutoff").template get<double>(),
-              optional_argument(adaptor_hypers)) {}
+              optional_argument_ghost(adaptor_hypers),
```

Comment:
adaptor_hypers might or might not have these arguments since they are optional. so I had to write some helper function to fit the code that was needed to do the cheks